apis/stackgres/v1/groupversion_info.go

```diff
@@ -0,0 +1,25 @@
+// +kubebuilder:object:generate=true
+// +groupName=vshn.appcat.vshn.io
```

Comment:
Shouldn't this be stackgres.io?

Or is it even necessary to have this comment here? We only need the deepcopy generation from the kubebuilder.

Reply:
Ah yes of course. Copy pasta error